packages/kas/src/compare.ts

```diff
@@ -16,18 +19,17 @@ export const compare = function (expr1, expr2, options) {
      */
 
     if (options !== undefined) {
-        // eslint-disable-next-line no-undef
-        options = _.extend(defaults, options);
+        options = {...defaults, ...options};
     } else {
         options = defaults;
     }
```

Comment:
FWIW, with spread syntax, we can just spread the `options` (even if they're `undefined`).

<img width="348" alt="image" src="https://github.com/user-attachments/assets/0a719b63-d6df-4ac3-ab14-0da79327472b">